.gitignore

```diff
@@ -18,8 +18,6 @@ target/
 *.iws
 *.iml
 *.ipr
-src/test/resources
-src/main/resources
```

Comment:
What is the reason to change it?

Reply:
App won't compile on git, was getting the "can't find can't find db.chagelog-master.yaml"  error